cmd/sealer/cmd/cluster/delete.go

```diff
@@ -69,14 +70,16 @@ func NewDeleteCmd() *cobra.Command {
 				mastersToDelete = deleteFlags.Masters
 				workersToDelete = deleteFlags.Nodes
 			)
-
-			if mastersToDelete == "" && workersToDelete == "" && !deleteAll {
-				return fmt.Errorf("you must input node ip Or set flag -a")
-			}
-
 			workClusterfile := common.GetDefaultClusterfile()
-			if deleteAll {
-				return deleteCluster(workClusterfile)
+			if deleteClusterFile == "" {
+				if mastersToDelete == "" && workersToDelete == "" && !deleteAll {
+					return fmt.Errorf("you must input node ip Or set flag -a")
+				}
```

Comment:
if mastersToDelete == "" && workersToDelete == "" && !deleteAll && deleteClusterFile =="{
				return fmt.Errorf("you must input node ip Or set flag -a")
			}